packages/eas-cli/src/commands/update/index.ts

```diff
@@ -604,6 +608,18 @@ export default class UpdatePublish extends EasCommand {
           ])
         );
         Log.addNewLineIfNone();
+        if (
+          isUploadedAssetCountAboveWarningThreshold(uploadedAssetCount, assetLimitPerUpdateGroup)
```

Comment:
If someone is above the max limit we should probably display a different message and throw an error, it would look a bit strange to display sth as a warning, and a few seconds later fail with an error for that reason(I'm assuming the error will be thrown latter when attempting to upload those assets)

Reply:
Looking into this, an earlier step in this process (`getUploadURLsAsync`) will explicitly fail with a descriptive error if there are too many assets. So we should only hit this line when `assets.length <= assetLimit`. I'll update the text to reflect that.